.github/workflows/unit_tests.yml

```diff
@@ -12,8 +12,6 @@ jobs:
         exclude:
           - os: macOS-latest
             python-version: "3.9"
-          - os: macOS-latest
-            python-version: "3.10"
```

Comment:
did you mean to drop 3.10 here?

Reply:
Drop from exclude list, which means we will conduct py310 unit test on macos

Reply:
oh i missed the `exclude` key. Nice.